addons/im_livechat/static/src/legacy/public_livechat_history_tracking.js

```diff
@@ -0,0 +1,22 @@
+odoo.define('im_livechat.legacy.im_livechat.HistoryTracking', function (require) {
+"use strict";
+
+var utils = require('web.utils');
+var { HISTORY_LIMIT, LIVECHAT_COOKIE_HISTORY } = require ("im_livechat.legacy.im_livechat.Constants");
```

Comment:
Have newline in-between, because JS modules in different odoo modules (`web` vs `im_livechat`), so different grouping.